backend/clap/proto/data_store.proto

```diff
@@ -137,15 +137,17 @@ message NewSubmissionRatingRequest {
 }
 
 message PartialRatingRequest {
-    int32 criterion_id      = 1;
-    int32 rating_id         = 2;
-    int32 partial_rating_id = 3;
-    string justification    = 4;
-    int32 points            = 5;
+    int32 assessor_id       = 1;
```

Comment:
this means that fiels `.pb.go`  in data-store are outdated. Please regenerate them:
```
cd zpi/backend/datastore
make generated 
```